cfgov/unprocessed/apps/mmt-my-money-calendar/js/views/wizard/beginning-balance-options.js

```diff
@@ -0,0 +1,89 @@
+import { observer } from 'mobx-react';
+import { useStore } from '../../stores';
+
+function BeginningBalanceOptions(props) {
+  const { InputWizardStore } = useStore();
+  console.log('props', props);
+
+  return (
+    <div className="expenses-options">
+      <form className="wizard-form">
+        <fieldset className="o-form_fieldset ">
+          <div className="m-form-field m-form-field__lg-target m-form-field__checkbox">
+            <input
+              className="a-checkbox"
+              type="checkbox"
+              id="Checking Account"
+              name="Checking Account"
+              onChange={props.handleCheckbox}
+              checked={Boolean(
+                InputWizardStore.selectedInputScreens.find((screen) => screen.category === 'Checking Account')
```

Comment:
Storing the selected screens as an array in this case is going to cause a lot of unnecessary iteration. It's probably not a huge deal for performance, but it might be better to store the selected screens as an object so you can look them up by key in an O(1) operation rather than O(n).